docs/content/operations/recommendations.md

```diff
@@ -5,6 +5,39 @@ layout: doc_page
 Recommendations
 ===============
 
+# Some Genearl guidelines
+
+JVM Flags..
+```
+-Duser.timezone=UTC
+-Dfile.encoding=UTF-8
+-XX:+PrintGCDetails
+-XX:+PrintGCDateStamps
+-XX:+PrintGCTimeStamps
+-XX:+PrintGCApplicationStoppedTime
+-XX:+PrintGCApplicationConcurrentTime
+-Xloggc:/var/logs/druid/historical.gc.log
+-XX:+UseGCLogFileRotation
+-XX:NumberOfGCLogFiles=50
+-XX:GCLogFileSize=10m
+-XX:+ExitOnOutOfMemoryError
+-XX:+HeapDumpOnOutOfMemoryError
+-XX:HeapDumpPath=/var/logs/druid/historical.hprof
+-XX:MaxDirectMemorySize=10240g
```

Comment:
I usually recommend people set `-Djava.io.tmpdir` explicitly since sometimes /tmp is not where they actually want to store tmp files.

Reply:
we have this set also, wasn't sure if its general enough... added with some explanation